examples/libsodium/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.0)
+project(libsodium-example)
+
+set(HUNTER_CONFIGURATION_TYPES Release) #Release or Debug
```

Comment:
note that it's not always debug+release, in general user can customize this stuff and scheme should loop over `HUNTER_PACKAGE_CONFIGURATION_TYPES`, but in practice the only one scheme do the right thing now, cmake-based: https://github.com/ruslo/hunter/blob/0380dbd81b80007f4c9c3d01e309d9820a833619/cmake/schemes/url_sha1_cmake.cmake.in#L82